packages/terra-status-view/src/StatusView.jsx

```diff
@@ -98,35 +98,42 @@ class StatusView extends React.Component {
    * with 40% of the remaining space at the top with 60% on the bottom.
    */
   showAndCenterItems() {
-    const viewHeight = this.innerNode.offsetHeight;
-
     let paddingTop = 0;
     let paddingBottom = 0;
     let showGlyph = false;
+    let newViewHeight = 0;
+    let contentHeight = 0;
+    let contentWithGlyphHeight = 0;
+
+    const viewHeight = this.innerNode.getBoundingClientRect().height;
+    newViewHeight = viewHeight;
 
     // get the content heights of the nodes that are to be shown if have content and cannot be hidden
-    let componentsHeight = this.titleNode.offsetHeight;
-    if (this.actionsNode) {
-      componentsHeight += this.actionsNode.offsetHeight;
-    }
-    if (this.messageNode) {
-      componentsHeight += this.messageNode.offsetHeight;
-    }
-    if (this.dividerNode) {
-      componentsHeight += this.dividerNode.offsetHeight;
+    const titleNodeHeight = this.titleNode.getBoundingClientRect().height;
+    const actionNodeHeight = this.actionsNode ? this.actionsNode.getBoundingClientRect().height : 0;
+    const messageNodeHeight = this.messageNode ? this.messageNode.getBoundingClientRect().height : 0;
+    const dividerNodeHeight = this.dividerNode ? this.dividerNode.getBoundingClientRect().height : 0;
+    contentHeight = (titleNodeHeight + actionNodeHeight + messageNodeHeight + dividerNodeHeight);
+
+    contentWithGlyphHeight = contentHeight + (this.glyphNode ? this.glyphNode.getBoundingClientRect().height : 0);
+    // if inner view node height is lesser than the components along with glyph inside the inner view.
+    // and the height difference is very less (i.e 0.0001) then add that comparision value to the
+    // the inner view height.
+    if (contentWithGlyphHeight - viewHeight <= 0.0001) {
```

Comment:
nit: want to create constant to store `0.0001`, then reuse it on the next two lines? something like `precision` or `roundingPrecision`